w04h03/test/pgdp/AdditionalQuickHullTests.java

```diff
@@ -0,0 +1,105 @@
+package pgdp.convexhull;
```

Comment:
Are you sure that this is the correct import (under the condition that this file is located under `w04h03/test/pgdp/AdditionalQuickHullTests.java`)?

I needed to change the package to the following line to get it to work.

```suggestion
package pgdp;
```